### ISSUE TITLE
Descriptive upload validation: Validate header path

### ISSUE THREAD

**Issue by justinlittman** [sul-dlss/argo#3441]
For individual and bulk descriptive upload, for each header other than "druid" and "source id", validate that header can resolve to a path in the cocina model. When there is a problem, report "Column header invalid: #{header}."

See `DescriptionImport` for how to map from header to path in cocina model.

refs #3375

**Comment by peetucket** [comment]
```
Cocina::Models::Description.schema.find { |key| key.name == :title }.type.type.type.member.schema.find { |key| key.name == :structuredValue }.type.type.type.member.schema.find {|key| key.name == :value }
```

**Comment by peetucket** [comment]
Wait for https://github.com/sul-dlss/argo/pull/3477 to be merged as you will be adding methods to a class getting expanded in that PR.

**Comment by peetucket** [comment]
See #3490 for the beginnings of this work extracted from #3477

**Comment by jcoyne** [comment]
Fixed by #3441
